docs/how-to.md

```diff
@@ -28,10 +28,6 @@ the following instructions can be used for every role (see [Use Case Diagrams](u
         - publish device model info
         - publish X509 certificates
         - revoke X509 certificates        
-    - [Test House](#test-house-instructions) 
```

Comment:
Please remove a reference to TestHouse from https://github.com/zigbee-alliance/distributed-compliance-ledger#instructions as well.